src/dev/tsconfig.prod.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "./tsconfig.app.json",
+  "compilerOptions": {
+    "paths": {
+      "@clr/angular": ["dist/clr-angular", "dist/clr-angular/*"],
```

Comment:
nit: we can just do `"@clr/*": ["dist/clr-*"]` I think?

Reply:
Was not able to get this syntax to work quite right but was able to remove the deep import that should not of been there on the dev app